std_msgs/Header header
float32[] cartesian_state